spec/factories/preferences.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: preferences
+#
+#  id          :bigint           not null, primary key
+#  name        :string           not null
+#  description :text             not null
+#  restriction :boolean          default(FALSE), not null
+#  user_id     :bigint           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_preferences_on_user_id  (user_id)
+#
+FactoryBot.define do
+  factory :preference do
+    name { "MyString" }
+    description { "MyText" }
+    restriction { false }
+    user { nil }
+  end
+end
```

Comment:
Great job! A realistic improvement suggestion would be to use the Faker gem to generate dynamic and test data for the name and description fields

```suggestion
FactoryBot.define do
  factory :preference do
    name { Faker::Name.unique.name }
    description { Faker::Lorem.sentence }
    restriction { Faker::Boolean.boolean }
    user { nil }
  end
end
```

Reply:
Great improvement and suggestion! I'll definitely keep the Faker gem in mind for future testing. It’s perfect for adding realistic test data quickly and reliably. Thanks for the tip!